### PR TITLE
Ignore all ns

### PR DESCRIPTION
**Pull request by qpdpQ**
fix for ticket: https://github.ibm.com/IBMPrivateCloud/roadmap/issues/58091#issuecomment-55546849
we should ignore the role for nss operator itself it in all namespaces

### CODE REVIEW

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Comment by Daniel-Fan** [comment]
@qpdpQ 
Could we try `make code-dev` to fix the format issue?

**Comment by Daniel-Fan** [comment]
Could we also update `  golangci-lint-version` to `1.51.x`? This is required for golang version `1.20` on prow cluster
https://github.com/IBM/ibm-namespace-scope-operator/blob/master/common/config/.golangci.yml#L3

**Comment by qpdpQ** [comment]
lint error
```
level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
level=warning msg="[runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner. Replaced by revive."
level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
level=warning msg="[runner] The linter 'interfacer' is deprecated (since v1.38.0) due to: The repository of the linter has been archived by the owner. "
level=warning msg="[runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
```
remove them

File: controllers/namespacescope_controller.go [hunk 1]
```diff
@@ -209,7 +210,6 @@ func (r *NamespaceScopeReconciler) UpdateConfigMap(instance *operatorv1.Namespac
 			if err := r.RestartPods(instance.Spec.RestartLabels, cm, instance.Namespace); err != nil {
 				return err
 			}
-			return nil
```
Comment (Daniel-Fan) [inline controllers/namespacescope_controller.go:212]: I think we could combine them together
```
return r.RestartPods(instance.Spec.RestartLabels, cm, instance.Namespace)
```

**Pull request merged**